tests/misc_env/install_prereq.py

```diff
@@ -94,6 +97,7 @@ def run(**kw):
 
 def install_prereq(
     ceph,
+    test_data,
```

Comment:
This changes the signature of the method. If it is not passed, we would need to set it to `None` by default.

Reply:
Sure, i will update.